test/e2e/e2e_test.go

```diff
@@ -125,3 +125,118 @@ func TestClusterHealthy(t *testing.T) {
 	// 'testEnv' is the env.Environment you set up in TestMain
 	_ = testEnv.Test(t, feature.Feature())
 }
+
+func TestScaling(t *testing.T) {
+	testCases := []struct {
+		name            string
+		initialSize     int
+		scaleTo         int
+		expectedMembers int
+	}{
+		{"ScaleDownFrom3To1", 3, 1, 1},
+		{"ScaleUpFrom1To3", 1, 3, 3},
```

Comment:
Also suggest to add field name
```suggestion
		{"ScaleInFrom3To1", 3, 1, 1},
		{"ScaleOutFrom1To3", 1, 3, 3},
```

Reply:
Changed to 
```
{name: "ScaleInFrom3To1", initialSize: 3, scaleTo: 1, expectedMembers: 1},
{name: "ScaleOutFrom1To3", initialSize: 1, scaleTo: 3, expectedMembers: 3},
``` 
https://github.com/etcd-io/etcd-operator/pull/192/files#diff-d5d12b6ea07b0a78c9ad1987e1486c565ecf16839f77a93e91b09475cf6c0e99R136-R137